### ISSUE TITLE
Suspected bug when <load addr> contains hex letters in -prg <app.prg>,<load addr>

### ISSUE THREAD

**Issue by mobluse** [commanderx16/x16-emulator#139]
I believe I found a bug when looking at the code in main.c. The following feature may not always work. I have not tested if it works or not.
```
-prg <app.prg>[,<load_addr>]
	Load application from the local disk into RAM
	(.PRG file with 2 byte start address header)
	The override load address is hex without a prefix.
```

snprintf(paste_text, sizeof(paste_text_data), "SYS$%04x\r", start);

I believe that x should be changed to X since PETSCII-UC only has uppercase letters.

In general I would prefer if uppercase hex digits were always printfed in e.g. log messages since they work in PETSCII-UC, and outputted hex numbers might end up on a display in PETSCII-UC mode.

**Comment by elsayed202** [comment]
i prefer it

**Comment by TowerZhao** [comment]
i prefer it

**Comment by liuxipeng158** [comment]
good

**Comment by liuxipeng158** [comment]
i prefer it

**Comment by mist64** [comment]
Fixed in 881c3ae
